src/Symfony/Component/Config/ValidatorConfigCacheFactory.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config;
+
+/**
+ * A ConfigCacheFactory implementation that validates the
+ * cache with an arbitrary set of metadata validators.
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+class ValidatorConfigCacheFactory implements ConfigCacheFactoryInterface
```

Comment:
Why creating a new config cache factory given adding validators to the old one is doable without BC break ? (But will prevent the deprecation due to https://github.com/symfony/symfony/pull/15692/files#diff-9f40590069e1fa6582e56bcea49e09a4R77)
Will the `ConfigCacheFactory` trigger a deprecation saying to use the new `ValidatorConfigCacheFactory` instead ?